src/compiler/resolver_method.cc

```diff
@@ -4808,6 +4808,15 @@ void MethodResolver::visit_call_primitive(ast::Call* node) {
     }
   }
 
+  if (false) {
```

Comment:
I should probably just drop this for now. I've been meaning to find a place - post type propagation - to check that methods that call primitives have a correct type annotation. Not entirely sure where to put it yet.